### ISSUE TITLE
Change width

### ISSUE THREAD

**Issue by bastifix** [abishek/fishbone-diagram-angular#6]
I haven't found a way to adjust the width of the diagram. Is it possible to change it, or even set it to responsive?

Works so nice so far, thanks very much!

**Comment by abishek** [comment]
I haven't really designed it to be responsive. Let me see if I can change that. The layout gets very messy in small screens 🤷‍♂️

**Comment by bastifix** [comment]
Of course, it doesn't have to be repsonsive (although it would be nice) - select an own width would let it be more flexible ;-)
